website/content/api-docs/acl/auth-methods.mdx

```diff
@@ -0,0 +1,392 @@
+    "CreateIndex": 12,
+    "ModifyIndex": 32
+}
+```
+
+## List Auth Methods
+
+This endpoint lists all ACL Auth Methods. This lists the auth methods that have
+been replicated to the region, and may lag behind the authoritative region.
+
+| Method | Path                | Produces           |
+| ------ | ------------------- | ------------------ |
+| `GET`  | `/acl/auth-methods` | `application/json` |
+
+The table below shows this endpoint's support for
+[blocking queries](/api-docs#blocking-queries),
+[consistency modes](/api-docs#consistency-modes) and
+[required ACLs](/api-docs#acls).
+
+| Blocking Queries | Consistency Modes | ACL Required                                                                                                                             |
+| ---------------- | ----------------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| `YES`            | `all`             | `management` for all auth methods.<br />Output when given a non-management token will be limited to the auth methods on the token itself |
+
+### Sample Request
+
+```shell-session
+$ curl \
+    --header "X-Nomad-Token: <NOMAD_TOKEN_SECRET_ID>" \
+    https://localhost:4646/v1/acl/auth-methods
+```
+
+### Sample Response
+
+```json
+[
+    {
+        "CreateIndex": 12,
+        "Default": true,
+        "ModifyIndex": 32,
+        "Name": "example-acl-auth-method"
+    }
+]
+```
+
+## Read Auth Method by Name
+
+This endpoint reads an ACL Auth Method with the given name. This queries the
+auth method that has been replicated to the region, and may lag behind the
+authoritative region.
+
+| Method | Path                            | Produces           |
+| ------ | ------------------------------- | ------------------ |
+| `GET`  | `/acl/auth-method/:method_name` | `application/json` |
+
+The table below shows this endpoint's support for
+[blocking queries](/api-docs#blocking-queries),
+[consistency modes](/api-docs#consistency-modes) and
+[required ACLs](/api-docs#acls).
+
+| Blocking Queries | Consistency Modes | ACL Required                              |
+| ---------------- | ----------------- | ----------------------------------------- |
+| `YES`            | `all`             | `management` or token with access to role |
+
+### Parameters
+
+- `:method_name` `(string: <required>)` - Specifies the name of the ACL Auth
+  Method. This is specified as part of the path.
+
+### Sample Request
+
+```shell-session
+$ curl \
+    --header "X-Nomad-Token: <NOMAD_TOKEN_SECRET_ID>" \
+    https://localhost:4646/v1/acl/auth-method/example-acl-auth-method
+```
+
+### Sample Response
+
+```json
+{
+    "MaxTokenTTL": "1h0m0s",
+    "Name": "example-acl-auth-method",
+    "Type": "OIDC",
+    "TokenLocality": "global",
+    "Default": true,
+    "Config": null,
+    "CreateTime": "0001-01-01T00:00:00Z",
+    "ModifyTime": "0001-01-01T00:00:00Z",
```

Comment:
I think it would be nice to have these as non-default time.Time values, to make it feel a little more real.